### ISSUE TITLE
Cant change wallet adresse on new gui

### ISSUE THREAD

**Issue by Casermandl** [mysteriumnetwork/node#5469]
i cant change my Wallet Adresse in the new gui. if i go to "settle now" i see my adress and if i click on the pencile on the right side the window closes. i cant change it.

**Comment by mdomasevicius** [comment]
Which browser are you using?
Does the browser has extensions?

**Comment by Casermandl** [comment]
yes, i don`t care about cookies and adblock but both are disabled for the avado/mysterium website
